db/migrate/20200511041621_create_posts.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePosts < ActiveRecord::Migration[6.0]
+  def change
+    create_table :posts do |t|
+      t.string :description
+      t.integer :user_id
```

Comment:
Please make sure to add migration for adding references for such columns

Reply:
Added the references for user_id column in post table